doc/apiary/v2/fiware-ngsi-simple.apib

```diff
@@ -50,16 +50,130 @@ to keep your client decoupled from implementation details.
 
 ## Context Entities [/v2/entities{?type,pattern,limit,offset}]
```

Comment:
Not sure of the parameters part has to be filled... Maybe it should be the union set of the required/optional parameters for all HTTP verbs (GET, POST, PUT, etc.) on the /v2/entities URL path?